packages/blade/src/components/QuickFilters/QuickFilter.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import type { QuickFilterProps, QuickFilterContentProps } from './types';
+import { useQuickFilterGroupContext } from './QuickFilterGroup';
+import { Card, CardBody } from '~components/Card';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+import { Radio } from '~components/Radio';
+import { Checkbox } from '~components/Checkbox';
+import { makeAnalyticsAttribute } from '~utils/makeAnalyticsAttribute';
+import { metaAttribute } from '~utils/metaAttribute';
+import BaseBox from '~components/Box/BaseBox';
+
+const QuickFilterContent = ({
+  value,
+  title,
+  trailingElement,
```

Comment:
```suggestion
  trailing,
```

In most of other components we just use `trailing` as the terminology.